### DISCUSSION TITLE
公告的使用有插件吗？

### DISCUSSION THREAD

**Discussion by erkang1** [vuepress-theme-hope/vuepress-theme-hope#2748] [Q&A] [answered]
公告配置能通过插件实现吗？

**Comment by erkang1** [comment]
![image](https://user-images.githubusercontent.com/51319096/216875291-9f9f3ca9-c1b1-460c-a500-c93092dd4d76.png)

**Reply by Mister-Hope** [reply]
https://plugin-components.vuejs.press/zh/guide/notice.html

**Comment by Mister-Hope** [comment, marked as answer]
https://plugin-components.vuejs.press/zh/guide/notice.html
